apps/examples/wifi_manager/wm_test/wm_test.c

```diff
@@ -843,10 +884,53 @@ int _wt_parse_dns(struct wt_options *opt, int argc, char *argv[])
 	return 0;
 }
 
-int _wt_parse_commands(struct wt_options *opt, int argc, char *argv[])
+int _wt_parse_connectbyrssi(struct wt_options* opt, int argc, char* argv[])
```

Comment:
This function use 5 argv array. Could you add description of argv array for better understanding?

Reply:
Done